Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -363,7 +363,7 @@ - (void)setUniversalLinkUrl:(NSString *)universalLinkUrl {
 }
 
 - (NSString *)sessionParams {
-    if (_sessionParams) {
+    if (!_sessionParams) {
```

Comment:
But if the app crashes, won't we create a new session and the session params will be overwritten? Probably I am confused because I am not clear why we save session params on disk. I need to dig deeper in the code to understand the session management part.